src/wazuh_modules/syscollector/syscollector_bsd.c

```diff
@@ -219,13 +219,15 @@ char* sys_parse_pkg(const char * app_folder, const char * timestamp, int random_
         cJSON_AddStringToObject(object, "timestamp", timestamp);
         cJSON_AddItemToObject(object, "program", package);
         cJSON_AddStringToObject(package, "format", "pkg");
-
+        int void_file=1;
```

Comment:
Here we should check if the `Info.plist` file is valid to be parsed, not when looking for the name program inside the file.